src/posit/connect/context.py

```diff
@@ -0,0 +1,43 @@
+import functools
+from typing import Optional, Protocol
+
+
+def requires(version: str):
+    def decorator(func):
+        @functools.wraps(func)
+        def wrapper(instance: ContextManager, *args, **kwargs):
+            ctx = instance.ctx
+            if ctx.version and ctx.version < version:
```

Comment:
Need to update this with better version comparison using `packaging.version.Version` or something else.